app/models/person.rb

```diff
@@ -3,17 +3,21 @@
 # Represents an author of a publication
 class Person < ApplicationRecord
   has_and_belongs_to_many :publications
-  store_accessor :metadata, :name, :institutionalAffiliation, :department
+  store_accessor :metadata, :name, :institutionalAffiliations, :departments
 
-  delegate :name, :country, to: :institution_entity, prefix: :institution
+  delegate :name, :country, to: :institution_entities, prefix: :institutions
 
-  def institution_entity
-    @institution_entity ||= Organization.find_by(uri: institutionalAffiliation)
+  def institution_entities
+    @institution_entities ||= institutionalAffiliations.map do |institutional_affiliation|
```

Comment:
How about `Organization.where(uri: institutionalAffiliations)` instead. I think this is a little easier to understand and a little more idiomatic for Rails.

Reply:
It should also mean fewer SQL queries. :+1: